app/assets/stylesheets/base/_buttons.scss

```diff
@@ -75,8 +75,8 @@ button[type="submit"],
 }
 
 .btn-small {
-  padding: ($small-spacing/2);
-  font-size: 12px;
+  padding: ($small-spacing/2) !important;
```

Comment:
My bet is because the kind of tag, link_to generate a `<a/>` tag, and f.submit generate a input submit, but the both are included on sass rules, I can figure out why just one works... (I noticed because the "Editar" button are working as expected, but the `f.submit` not, with the same classes!!)